### ISSUE TITLE
Fix Request for AvoidUnnecessaryStringBuilderCreation: false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#195]
appends are on 1 line, but the forEach is like a loop, and the toString is not on the same line.
```
void foo(MethodArgumentNotValidException e) {
   StringBuilder stringBuilder = new StringBuilder();
   e.getBindingResult().getFieldErrors().forEach(fieldError -> stringBuilder.append(fieldError.getField()).append(" ").append(fieldError.getDefaultMessage()).append(". "));
   errorLogger.logUnexpectedError(stringBuilder.toString())
}
```
